java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S2142_java.html

```diff
@@ -2,6 +2,10 @@
 Instead, <code>InterruptedExceptions</code> should either be rethrown - immediately or after cleaning up the method's state - or the method should be
 reinterrupted. Any other course of action risks delaying thread shutdown and loses the information that the thread was interrupted - probably without
 finishing its task.</p>
+<p>Similarly, the <code>ThreadDeath</code> exception should also be propagated. According to its JavaDoc:</p>
```

Comment:
This change has been postponed in 5.x: See [SONARJAVA-2255](https://jira.sonarsource.com/browse/SONARJAVA-2255)